monai/data/utils.py

```diff
@@ -399,3 +401,40 @@ def create_file_basename(postfix, input_file_name, folder_path, data_root_dir=""
 
     # add the sub-folder plus the postfix name to become the file basename in the output path
     return os.path.join(subfolder_path, filename + "_" + postfix)
+
+
+def compute_importance_map(patch_size, mode='constant', sigma_scale=0.125, device=None):
+    """Get importance map for different weight modes.
+
+    Args:
+        patch_size (tuple): Size of the required importance map. This should be either H, W [,D].
+        mode (str): Importance map type. Options are 'constant' (Each weight has value 1.0)
+            or 'gaussian' (Importance becomes lower away from center).
+        sigma_scale (float): Sigma_scale to calculate sigma for each dimension 
```

Comment:
I think you didn't use this `sigma_scale` in the sliding window program, should we keep it?

Reply:
It is used in gaussian in L425

Reply:
I mean seems when you call this function in sliding window, you didn't set this parameter.
You want to keep it for other use cases that may need to change this sigma_scale?
Thanks.

Reply:
Yes I'm trying to keep utils function configurable.